probe.go

```diff
@@ -630,6 +629,47 @@ func probeHAStatistics(c FortiHTTP) ([]prometheus.Metric, bool) {
 	return m, true
 }
 
+func probeLicenseStatus(c FortiHTTP) ([]prometheus.Metric, bool) {
+	var (
+		vdomUsed = prometheus.NewDesc(
+			"fortigate_license_vdom_usage",
+			"Info metric regarding used vdom licenses",
```

Comment:
Update the description of these metrics. Maybe something like "The amount of VDOM licenses currently used"  and "The total amount of VDOMs licenses available"

Reply:
Yeah, that's a better wording, thanks :)